sdk/agent/agent.go

```diff
@@ -287,7 +287,7 @@ func (a *Agent) Open() error {
 		ObservationPeriodTime:      a.observationPeriodTime,
 		ObservationPeriodLedgerGap: a.observationPeriodLedgerGap,
 		Asset:                      "native",
-		ExpiresAt:                  time.Now().Add(a.maxOpenExpiry),
+		ExpiresAt:                  time.Now().Add(a.maxOpenExpiry / 2),
```

Comment:
This probably should have gone into a separate PR, but I had to make the change to make the console app work. If a participant creates an open with the expiry set to the max open expiry and both participants have the same max open expiry, then if the other participants clock is slightly faster, the other participant will reject the open because it's outside their expiry. A simple way to avoid clock drift is to select a expiry half way between the participants max open expiry.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/370